data/json/items/discs.json

```diff
@@ -0,0 +1,802 @@
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_ogre2",
+    "name": { "str": "DVD disc - Shrek 2" },
+    "description": "A DVD containing the second installment of the Shrek movie series, Shrek 2, in which the adventures of Shrek and Princess Fiona continue as they visit her royal parents, facing challenges due to their unconventional marriage and discovering family secrets.  It is rated PG and is from 2004.",
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_ogre3",
+    "name": { "str": "DVD disc - Shrek the Third" },
+    "description": "As the name says, this DVD contains the third installment of the Shrek movie series.  It follows Shrek as he navigates the challenges of ruling the kingdom.  When King Harold falls ill and perishes, Shrek sets out to find the rightful heir, encountering characters and many challenges along the way.  It is rated PG and is from 2007.",
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_ogre4",
+    "name": { "str": "DVD disc - Shrek Forever After" },
+    "description": "A DVD that contains the fourth installment of the Shrek movie series.  This movie explores an alternate reality created when Shrek makes a deal with Rumpelstiltskin, leading to a world where he never existed.  Shrek must undo the consequences of these actions and save his friends, family, and future.  It is rated PG and is from 2010.",
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_lion",
+    "name": { "str": "DVD disc - The Lion King" },
+    "description": "A DVD that contains the tale of Simba, a young lion prince, who faces the challenges of reclaiming his kingdom after the murder of his father, Mufasa.  Guided by the lessons of his journey, Simba learns the circle of life and his rightful place as king.  This film is a classic animated adventure and is rated G, suitable for all audiences (not that there are any nowadays), and is from 1994.",
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
+  {
+    "type": "GENERIC",
+    "id": "dvd_foodperson",
+    "name": { "str": "DVD disc - Foodperson: The Marketable Movie" },
+    "description": "Foodplace presents this 94.3% watchable motion picture DVD.  It stars Foodperson as they attempt to retrieve the Object of Plot Relevance from the possession of the Antagonist.  Rated D for delicious and is from 2018.",
+    "copy-from": "dvd_blank",
+    "price": 500,
+    "price_postapoc": 75
+  },
```

Comment:
```suggestion
```
A whole trilogy of movies about a fast food mascot doesn't make sense. This too far beyond the ridiculous but maybe tongue in cheek marketing of foodplace.

Reply:
We have damn loyalty cards for it so this one is acceptable I think

Reply:
Mac and Me is a real movie centered around aliens who survive on Coca Cola with a , fast food places funding a movie based on their mascot isn't too far fetched imo.

Reply:
Mac and Me is a real movie about aliens who survive off Coca Cola with a full musical number in McDonald's where the main character is named after a sandwich, I don't think a movie based on a fast food mascot is too far removed from reality imo.